frontend/packages/console-plugin-sdk/src/codegen/plugin-resolver.ts

```diff
@@ -68,7 +68,7 @@ export const getMonorepoRootDir = () => {
   return findUp.sync(
     (currentDir) => {
       return fs.existsSync(path.join(currentDir, 'package.json'))
-        ? readPkg.sync({ cwd: currentDir, normalize: true }).name === '' && currentDir
+        ? readPkg.sync({ cwd: currentDir, normalize: true }).name === 'openshift-console' && currentDir
```

Comment:
```suggestion
        ? readPkg.sync({ cwd: currentDir, normalize: true }).name === 'openshift-console' && 
        currentDir
```
Please run prettier on this file. Prettier tests are failing.